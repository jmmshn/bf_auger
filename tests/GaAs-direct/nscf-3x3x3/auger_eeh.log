 ======== Parameters ME_INP =========
 nb=          20
 nr1,  nr2,  nr3  =    0   0   0
 cEIGfile  = GaAs.eig                                          
 isoc  =            1
 ivbm  =            9
 Egap_min  =    1.00000000000000       3.674930882447527E-002
 Egap_step  =   1.000000000000000E-002  3.674930882447527E-004
 epsilon_0  =   0.000000000000000E+000
 nelec  =            0
 ====================================
