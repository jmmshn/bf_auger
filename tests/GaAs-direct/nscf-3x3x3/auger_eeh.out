setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
 Number of irreducible kpts =           27
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
setting up k-point grids
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
setting up k-point grids
elec grid:    3   3   3
elec grid:    3   3   3
setting up k-point grids
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
setting up k-point grids
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
setting up k-point grids
elec grid:    3   3   3
setting up k-point grids
setting up k-point grids
elec grid:    3   3   3
setting up k-point grids
elec grid:    3   3   3
setting up k-point grids
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
setting up k-point grids
elec grid:    3   3   3
setting up k-point grids
setting up k-point grids
setting up k-point grids
elec grid:    3   3   3
elec grid:    3   3   3
setting up k-point grids
elec grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
setting up k-point grids
setting up k-point grids
elec grid:    3   3   3
setting up k-point grids
elec grid:    3   3   3
elec grid:    3   3   3
setting up k-point grids
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
setting up k-point grids
elec grid:    3   3   3
setting up k-point grids
elec grid:    3   3   3
elec grid:    3   3   3
setting up k-point grids
elec grid:    3   3   3
setting up k-point grids
setting up k-point grids
elec grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
elec grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
hole grid:    3   3   3
           1  of            1
 ===============================================
 JOB FINISHED
 ===============================================
